python/ray/autoscaler/updater.py

```diff
@@ -204,7 +206,10 @@ def do_sync(remote_path, local_path, allow_non_existing_paths=False):
             with cli_logger.group(
                     "Processing worker file mounts",
                     _numbered=("[]", previous_steps + 2, total_steps)):
+                cli_logger.print("synced files: {}",
+                                 str(self.cluster_synced_files))
                 for path in self.cluster_synced_files:
+                    path = os.path.expanduser(path)
```

Comment:
Can we pull this out into a separate PR and do it in the intializer (keep it consistent with the `self.file_mounts` treatment?